libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -238,11 +239,12 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
   }
 
   /**
-   * The array of functions to call against each search result in order
-   * to filter the search results when using the default search function. When
-   * using the `search` property to specify a custom search function, you must
+   * The array of functions to call against each search result. This filters
+   * the search results when using the default search function.
```

Comment:
```suggestion
   * the search results when using the `data` input and the default search function.
```
Should this have the same verbiage that we added to the other descriptions? It looks like this has a `data` input, so _I think_ so, but I'm not sure.